src/app/components/shared/website-capability-warning/website-status-warning.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { WebsiteStatusService } from "@baw-api/website-status/website-status.service";
+import { websiteStatusMenuItem } from "@components/website-status/website-status.menu";
+import { withUnsubscribe } from "@helpers/unsubscribe/unsubscribe";
+import { WebsiteStatus } from "@models/WebsiteStatus";
+import { takeUntil } from "rxjs";
+
+@Component({
+  selector: "baw-website-status-warning",
+  template: `
+    <div *ngIf="!isCapabilityHealthy" class="alert alert-danger">
+      The ability to {{ websiteCapabilityText }} is temporarily unavailable.
+      Please try again later.
+
+      <div>
+        <a [strongRoute]="websiteStatusRoute">More Information</a>
+      </div>
+    </div>
+  `,
+})
+export class WebsiteStatusWarningComponent
+  extends withUnsubscribe()
+  implements OnInit
+{
+  public constructor(private api: WebsiteStatusService) {
+    super();
+  }
+
+  @Input({ required: true }) public feature: keyof WebsiteStatus;
+  @Input({ required: true }) public websiteCapabilityText;
+  protected isCapabilityHealthy = true;
+  protected websiteStatusRoute = websiteStatusMenuItem.route;
+
+  public ngOnInit(): void {
+    this.api
+      .show()
+      .pipe(takeUntil(this.unsubscribe))
+      .subscribe((status: WebsiteStatus) => {
+        this.isCapabilityHealthy = status[this.feature] as boolean;
+      });
```

Comment:
potentially the same point about updates as with the indicator, but it's less important here since this component is reloaded on navigation. Still a user could be on a page for a while ...

Reply:
For these warning banners, I can extend the interval to 2 minutes.

Additionally, because we are fetching the status every 30 seconds, we only need to invalidate the cache on memoized cache on the status icon request. The warning banner should be able to use the memoized value of the status icon request (meaning that we are not making multiple requests every 2 minutes).